index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <title>Portfolio David Tamayo</title>
+</head>
+<body>
+    <div class="mainpage">
+        <header class="toolbar">
+            <h2 class="welcome">Welcome</h2>
+            <button class="vector">☰</button>
```

Comment:
- [ ] Your hamburger button should be inside a `nav` tag for the semantic, convention, and user experience for mobile device reasons. Please fix it :smile: